app/main.py

```diff
@@ -1 +1,60 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            food_points = self.appetite
+            self.appetite = 0
+            self.is_hungry = False
+            return food_points
+        return 0
+
+
+class Cat(Animal):
+    def __init__(
+            self,
+            name: str,
+            appetite: int = 3,
+            is_hungry: bool = True
+    ) -> None:
+
+        self.name = name
+        super().__init__(name, appetite)
+        self.is_hungry = is_hungry
```

Comment:
```suggestion
        super().__init__(name, appetite, is_hungry)
```
Can you explain why you need these lines?
Fix into Dog also